rpm/0001-Tracker-config-overrides.patch

```diff
@@ -1,7 +1,7 @@
-From aae99517e9159f9eb7980fb1e7e53606c8b8a04f Mon Sep 17 00:00:00 2001
+From c48ac84bc9c8b591d99fb1091e58dccc0d7fa710 Mon Sep 17 00:00:00 2001
 From: Islam Amer <islam.amer@jollamobile.com>
 Date: Wed, 12 Nov 2014 18:10:53 +0200
-Subject: [PATCH 1/7] Tracker config overrides
+Subject: [PATCH 1/5] Tracker config overrides
```

Comment:
It's a good habit to use --no-numbered (-N) when formatting patches. Similarly --zero-commit is good habit.

Reply:
Sure. Regenerated the patches.